PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -198,15 +203,24 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         electronVeto = Var("passElectronVeto()",bool,doc="pass electron veto"),
         pixelSeed = Var("hasPixelSeed()",bool,doc="has pixel seed"),
```

Comment:
can we improve the self documentation ? i.e. the doc string

electronVeto : to add "analysis not sensitive to electron -> photon fake rate"
pixelSeed: to add "analysis sensitive to electron -> photon fake rate"

see information in the wiki https://twiki.cern.ch/twiki/bin/viewauth/CMS/CutBasedPhotonIdentificationRun2